charts/operator/templates/clusterrole.yaml

```diff
@@ -286,7 +287,7 @@ rules:
     - patch
     - update
 {{- end -}}
-{{- if and .Values.rbac.create (and ( eq .Values.scope "Namespace" ) .Values.rbac.createAdditionalControllerCRs ) -}}
```

Comment:
`createAdditionalControllerCRs` is removed because Redpanda chart required cluster role?

Reply:
No, it was moved down, i added a new clusterrole that is deployed by default. This role is required to create rbac roles that redpanda chart needs. The rbac roles are the same as found in the redpanda/templates/rbac.yaml, that will answer your questions.